server/docs/getting-started.md

```diff
@@ -2,106 +2,251 @@
 3. When is the game over?
 4. What happens to the stake when the game is over?
 
-The contract has already been written, and can be found [here](https://github.com/counterfactual/monorepo/blob/master/packages/apps/contracts/HighRollerApp.sol).
+In this guide, we've provided you with a deployed contract, [HighRoller.sol](https://github.com/counterfactual/monorepo/blob/master/packages/apps/contracts/HighRollerApp.sol), to use for your dapp. Take a quick look at **HighRoller.sol** and see if you can find how it answers questions 1-4. We'll also run through this together in the next [section](https://github.com/counterfactual/website/blob/joey-editing/server/docs/getting-started.md#a-quick-look-at-the-contract).
 
-The UI and client logic for the game is implemented in **HighRoller.js**. Functionality handled includes:
+
+The UI and client logic for the game will be implemented in **app.js**. Functionality handled includes:
 
 1. Proposing a game to another user
 2. Accepting a proposal from another user
 3. Taking an action when it is your turn
 4. Listening for other players to take their turns
 5. Leaving a game when it’s over
 
-You will create this file and all the functionality by following this guide.
 
-Counterfactual nodes implement the interactions between these two components
+### In this Getting Start Guide, you’ll learn how to:
 
-dapp UI ( HighRoller.js ) < -- 1 -- > Counterfactual Node < -- 2 -- > GameLogic (HighRoller.sol)
+1. Create a new Counterfactual project repo
+1. Create an `AppFactory` instance, which will specify the application logic for **HighRoller**
+1. Use the `AppFactory`'s method `proposeInstallVirtual()` to propose a new virtual state channel when the user wants to play **HighRoller**
+1. Use the Counterfactual `NodeProvider`'s method `on()` to listen for the second player to accept the proposed virtual install
+1. Use the Counterfactual `NodeProvider`'s method `on()` to listen for updated state in the virtual channel
+1. Use the `AppInstance`'s `takeAction()` method to [update state](https://specs.counterfactual.com/en/latest/01-app-definition.html#progressing-state) in the virtual channel
+1. Use the `AppInstance`'s `uninstall()` method to close and resolve the virtual channel
 
-in three ways:
+Along the way, we'll see how the Counterfactual framework ensures that state is updated safely and securely. 
 
-1. **Starting a channel** - When users request / agree to play a game, the UI passes the request (via connection 1) to the Counterfactual node. The node then instantiates a state channel based on the game logic described by the contract (connection 2). Any staked funds the state channel requires are drawn from the Counterfactual wallet inside MetaMask.
+One more thing: to streamline your first dapp, we've built and deployed a bot that will accept `proposeInstallVirtual()` requests for **HighRoller** and play the game with your user. This means you only have to code the UI for the first player of **HighRoller**.
```

Comment:
Maybe add a footnote to this document called "How does the bot work?" and link to it here? It's pretty straightforward to explain once the reader knows what `on()` is and so on, so I think it would be helpful.

Reply:
We can also make this point more clearer in English by saying "obviously state channels require a counterparty and so for the sake of this getting started guide we have a bot that you can open a channel with"?